src/components/qas/questions/qaCalculators/shared/equation/FindKineticEnergy.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { addTheCommas, formatValue } from '../../../../../../lib/utilities';
+import {
+  equation,
+  fraction,
+  denominator,
+  exponent,
+} from './equation.module.scss';
+
+export default function FindKineticEnergy(props) {
```

Comment:
I really like the way the other calculators (for instance [the SizeCalc seen here](http://localhost:8000/identifying-pha-2/)) color (uses `$basePrimary`) the variables the user is providing values for (as well as the `?`).  Can we pull that styling over to this component as well?